app/src/main/java/app/mywatchlist/screens/MyWatchlistScreen.kt

```diff
@@ -0,0 +1,33 @@
+package app.mywatchlist
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.navigation.NavController
+import androidx.navigation.compose.rememberNavController
+
+@Composable
+fun MyWatchlistScreen(navController: NavController){
```

Comment:
Would just go with `WatchlistScreen`, the `my` is kind of redundant and only necessary for the app name.